### PR TITLE
Update Tdarr to version 2.00.21

### PR DESCRIPTION
**Pull request by dlsniper**
This updates Tdarr to version 2.00.21

### CODE REVIEW

File: catalog.json [hunk 1]
```diff
@@ -2041,9 +2041,9 @@
             "home": "https://home.tdarr.io/",
             "location": "/__w/charts/charts/community/tdarr",
             "latest_version": "1.0.12",
-            "latest_app_version": "2.00.20.1",
```
Comment (stavros-k) [inline catalog.json:2044]: Please revert the changes in `catalog.json`

File is updated automatically after merge.
Thanks

**Review by stavros-k — CHANGES_REQUESTED** [review]
Please see my comment regarding `catalog.json` file.

Additionally I've opened an issue upstream to kindly request to make versioning scheme consistent in order for the automations to pick up newer versions automatically.

Thanks

File: catalog.json (hunk 1 above)
Comment (dlsniper) [reply catalog.json]: Done, thank you for the quick review!

**Review by stavros-k — APPROVED** [review]

File: library/ix-dev/community/tdarr/Chart.yaml [hunk 2]
```diff
@@ -5,7 +5,7 @@ annotations:
 type: application
 version: 1.0.12
```
Comment (stavros-k) [inline library/ix-dev/community/tdarr/Chart.yaml:6]:
```suggestion
version: 1.0.13
```
